src/main/resources/wa-task-permissions-civil-civil.dmn

```diff
@@ -1028,13 +1028,13 @@ else
       <rule id="DecisionRule_0nkyesz">
         <description>Listing Officer task permissions</description>
         <inputEntry id="UnaryTests_11t87u1">
-          <text>"ClaimSettledRemoveHearing","ClaimDiscontinuedRemoveHearing"</text>
+          <text>"ClaimSettledRemoveHearing"</text>
         </inputEntry>
         <inputEntry id="UnaryTests_0li86qd">
           <text></text>
         </inputEntry>
         <inputEntry id="UnaryTests_1psba2h">
-          <text>"SD"</text>
+          <text>-</text>
```

Comment:
is this removed on purpose? or would it be put back in before merging to master?